apps/hash-frontend/src/pages/shared/block-collection/mention-view/mention-display.tsx

```diff
@@ -298,7 +299,7 @@ export const MentionDisplay: FunctionComponent<MentionDisplayProps> = ({
           }}
         >
           <Link
-            href={entityType?.schema.$id ?? "#"}
+            href={generateLinkParameters(entityType?.schema.$id).href || "#"}
```

Comment:
Why do we need to do this if the link component handles it for us?

```suggestion
            href={entityType?.schema.$id|| "#"}
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/3572/commits/b31daf5c29c4daa2fcd9729c030f2852da744120